WordPress/src/main/java/org/wordpress/android/ui/posts/GutenbergWarningFragmentDialog.kt

```diff
@@ -0,0 +1,135 @@
+package org.wordpress.android.ui.posts
+
+import android.app.Dialog
+import android.content.Context
+import android.os.Bundle
+import android.support.v7.app.AppCompatDialogFragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.CheckBox
+import org.wordpress.android.R
+import org.wordpress.android.widgets.WPTextView
+
+/**
+ * Basic dialog fragment with support for 1,2 or 3 buttons.
+ */
+class GutenbergWarningFragmentDialog : AppCompatDialogFragment() {
+    private lateinit var mTag: String
```

Comment:
Similar to an earlier comment, do we need to keep a tag around for the dialog?

Reply:
addressed as well in dd1c456